android/assets/jsons/Civ V - Vanilla/Policies.json

```diff
@@ -38,7 +38,7 @@
 			},
 			{
 				"name": "Tradition Complete",
-				"uniques": ["+[15]% growth [in all cities]","[+2 Food] [in all cities]"]
+				"uniques": ["+[15]% growth [in all cities]","Immediately creates a free [aqueduct] in each of your first [4] cities for free"]
```

Comment:
double 'free'

Reply:
and "aqueduct" should start with a capital letter.